fe/fe-core/src/main/java/com/starrocks/common/util/RuntimeProfile.java

```diff
@@ -261,14 +354,24 @@ private void printChildCounters(String prefix, String counterName, StringBuilder
         if (childCounterMap.get(counterName) == null) {
             return;
         }
-        Set<String> childCounterSet = new TreeSet<>(childCounterMap.get(counterName));
-
-        for (String childCounterName : childCounterSet) {
-            Counter counter = this.counterMap.get(childCounterName);
-            Preconditions.checkState(counter != null);
-            builder.append(prefix).append("   - ").append(childCounterName).append(": ")
-                    .append(printCounter(counter.getValue(), counter.getType())).append("\n");
-            this.printChildCounters(prefix + "  ", childCounterName, builder);
+        List<String> childNames = Lists.newArrayList(childCounterMap.get(counterName));
+
+        // Keep MIN/MAX metrics head of other child counters
+        List<String> reorderedChildNames = Lists.newArrayList();
```

Comment:
Lists.newArrayListWithCapacity()
```suggestion
        List<String> reorderedChildNames =  Lists.newArrayListWithCapacity(childNames)
```

Reply:
done